timesketch/frontend-ng/src/views/Studio.vue

```diff
@@ -0,0 +1,112 @@
+<!--
+Copyright 2019 Google Inc. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <div>
+    <!-- Left panel -->
+    <v-navigation-drawer app permanent :width="navigationDrawer.width" hide-overlay ref="drawer">
+      <v-toolbar flat>
+        <v-avatar class="mt-2 ml-n4">
+          <router-link to="/">
+            <v-img src="/dist/timesketch-color.png" max-height="25" max-width="25" contain></v-img>
+          </router-link> </v-avatar
+        ><b>Studio</b>
+        <v-spacer></v-spacer>
+      </v-toolbar>
+      <v-divider></v-divider>
+      <ts-sigma-rules></ts-sigma-rules>
+    </v-navigation-drawer>
+    <ts-sigma-rule-modification app @cancel="formatXMLString = false" :rule_uuid="id" v-if="type === 'sigma'">
+    </ts-sigma-rule-modification>
+    <ts-search-template-modification app :id="id" v-if="type === 'searchtemplate'"> </ts-search-template-modification>
+  </div>
+</template>
+
+<script>
+import TsSigmaRules from '../components/LeftPanel/SigmaRules'
+import TsSigmaRuleModification from '../components/Studio/SigmaRuleModification.vue'
+export default {
+  props: ['showLeftPanel', 'id', 'type'],
+  components: {
+    TsSigmaRules,
+    TsSigmaRuleModification,
+  },
+  data() {
+    return {
+      showSketchMetadata: false,
+      navigationDrawer: {
+        width: 450,
+      },
+      sketchId: 1, // TODO: this must be removed
+    }
+  },
+  created: function () {
+    this.$store.dispatch('updateSigmaList')
+    document.title = 'Timesketch Studio'
+  },
+  updated() {
+    this.$nextTick(function () {
+      this.setDrawerBorderStyle()
+      this.setDrawerResizeEvents()
+    })
+  },
+  computed: {
+    meta() {
+      return this.$store.state.meta
+    },
+  },
+  methods: {
+    setDrawerBorderStyle() {
+      let i = this.$refs.drawer.$el.querySelector('.v-navigation-drawer__border')
+      i.style.cursor = 'ew-resize'
+    },
+    setDrawerResizeEvents() {
```

Comment:
Oof, I have no idea what's going on here but I'm sure you can use vuetify's scaffolding CSS to accomplish this: https://vuetifyjs.com/en/getting-started/wireframes/

Reply:
straight from here: https://github.com/google/timesketch/blob/90f7b9fb70f45dfab024c371bf3bf9e30d2f9f5d/timesketch/frontend-ng/src/views/Sketch.vue#L324